rust/firezone-cli-utils/src/lib.rs

```diff
@@ -24,15 +24,18 @@ where
 /// Arguments common to all Firezone CLI components.
 #[derive(Args, Clone)]
 pub struct CommonArgs {
-    /// Firezone admin portal websocket URL
     #[arg(
         short = 'u',
         long,
-        env = "PORTAL_URL",
+        hide = true,
+        env = "API_URL",
         default_value = "wss://api.firezone.dev"
     )]
-    pub portal_url: Url,
+    pub api_url: Url,
+    /// Identifier generated by the portal to identify websocket connection.
```

Comment:
```suggestion
    /// Identifier generated by the portal to identify and display the device.
```

I think the `websocket` here might be too much information for the user.